### ISSUE TITLE
landing page for DSCensor

### ISSUE THREAD

**Issue by adf-ncgr** [LegumeFederation/legfed-django-site#30]
Technically, we already have a landing page for DSCensor:
https://www.legumefederation.org/dscensor/
but, before we make it "really" live (ie linked from somewhere), we might want to think about providing some helpful instructional materials (which appear to be "coming soon!")

it might be nice if all things considered to be "core federation" followed some similar approach, though not sure it's worth effort to reengineer anything for the sake of "mere consistency"... but see #29 anyway

**Comment by ctcncgr** [comment]
Hey Andrew,

There are a few things that should probably happen, but yes I think some help text or instructions would be useful.

I'm going to add a couple panels describing how to use the tool and put a footer and stuff on the page.

I think I'm going to try to source style from LegFed and see how it goes to try and make the feel more seamless.

**Comment by adf-ncgr** [comment]
agreed that there are a few TODOs outside of help text. add issues as you see fit (and self-assign where appropriate, which is probably almost always!) this was mostly just an afterthought on the conversation @jd-campbell and I had regarding #29
